src/test/java/com/ays/common/model/AysPhoneNumberBuilder.java

```diff
@@ -10,16 +10,16 @@ public AysPhoneNumberBuilder() {
 
     public AysPhoneNumberBuilder withValidFields() {
         return new AysPhoneNumberBuilder()
-                .withCountryCode(90L)
-                .withLineNumber(AysRandomUtil.generateNumber(10));
+                .withCountryCode("90")
+                .withLineNumber(AysRandomUtil.generateNumber(10).toString());
     }
 
-    public AysPhoneNumberBuilder withCountryCode(Long countryCode) {
+    public AysPhoneNumberBuilder withCountryCode(String countryCode) {
         data.setCountryCode(countryCode);
         return this;
     }
 
-    public AysPhoneNumberBuilder withLineNumber(Long lineNumber) {
+    public AysPhoneNumberBuilder withLineNumber(String lineNumber) {
```

Comment:
Bu sınıf içerisindeki geliştirmelere ihtiyaç kalmadığı için geri alabilir miyiz?